uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Form/AbsenceType.tpl

```diff
@@ -320,14 +320,28 @@
                   return hours_unit_value;
                 }
 
+                /**
+                 * Adds <input type="hidden"> before disabled <select>
+                 * with the same name and value to support value submission
+                 *
+                 * @param {jQueryElement} $select - the selector to be amended
+                 */
+                function supportDisabledSelector ($select) {
+                  if ($select.attr('disabled')) {
+                    $select.before('<input type="hidden" name="' +
+                      $select.attr('name') + '" value="' +
+                      $select.val() + '" />');
+                  }
+                }
+
                 $(document).ready(function() {
                   initToilControls();
                   initCarryForwardControls();
                   initColorPicker();
                   initDeleteButton();
-                  initCalculationUnitControls()
+                  initCalculationUnitControls();
+                  supportDisabledSelector($('select[name=calculation_unit]'));
```

Comment:
Something like `supportValueSubmissionOfDisabledSelector()` would be more appropriate.

Reply:
Done, but slightly differently.